src/data/faq_de.json

```diff
@@ -408,6 +408,18 @@
                             "Laut Beschluss der Gesundheitsminister*innen von Bund und Ländern müssen Geimpfte oder Genesene, die eine Auffrischimpfung (Boosterimpfung) erhalten haben, bei einer 2G+-Regelung momentan keinen aktuellen Test vorlegen. Ab Version 2.18 der Corona-Warn-App wird der Nachweis-Status „2G+“ angezeigt, wenn ein Zertifikat einer Auffrischimpfung in der App abgelegt ist.",
                             "Siehe auch: <ul><li><a href='#admission_policy' target='_blank'>Grundlegendes zur Anzeige des Status-Nachweises</a></li><li><a href='#admission_policy_2gplus' target='_blank'>Was bedeutet konkret der 2G+-Status-Nachweis?</a></li><li><a href='#admission_policy_2gplus_not_shown' target='_blank'>Warum wird der 2G+ -Status-Nachweis nicht angezeigt, obwohl ich ein Impf- oder Genesenenzertifikat und einen negativen Test habe?</a></li><li><a href='#admission_policy_2gplus_howto' target='_blank'>Wie verwende ich die kombinierte 2G+-Anzeige?</a></li></ul>"
                         ]
+                    },
+                    {
+                        "title": "Was passiert, wenn die Namen meiner Zertifikate nicht exakt übereinstimmen?",
+                        "anchor": "cert_fault_tolerant_name_handling",
+                        "active": false,
+                        "textblock": [
+                            "Ab Version 2.19 der Corona-Warn-App ist die Zuordnung von Zertifikaten zu Personen fehlertoleranter gestaltet. Kleine Unterschiede in der Schreibweise von Vor- und Nachnamen werden ignoriert, sodass die Zertifikate trotzdem derselben Person automatisch zugeordnet werden.",
+                            "Die Corona-Warn-App erkennt, dass Zertifikate zu einer Person gehören, wenn die Geburtsdaten exakt übereinstimmen und: <ul><li>Vor- und Nachnamen exakt übereinstimmen,</li><li>mindestens ein Vorname und mindestens ein Nachname übereinstimmen oder</li><li>Vor- und Nachnamen vertauscht sind.</li></ul>",
+                            "Für den Fall, dass Ihr Name Umlaute enthält werden diese automatisch umgewandelt. Der akademische Doktortitel führt nun nicht mehr zur Unterscheidung von Namen.",

```

Comment:
"Für den Fall, dass Ihr Name Umlaute enthält werden diese automatisch umgewandelt."

In this sentence there is a change to using the 2. person plural, which is a break with the rest of the description. Also there is a comma missing.

Suggest changing to:
"Für den Fall, dass ein Name Umlaute enthält, werden diese automatisch umgewandelt."

Reply:
Changed it in https://github.com/corona-warn-app/cwa-website/pull/2526/commits/3d76e6f74e60bd94c4530492103dd75055b0f04d, thanks for the suggestion